test-common/utils.c

```diff
@@ -2972,7 +2972,7 @@ eos_test_client_prepare_volume (EosTestClient *client,
                                                                            "libeos-updater-util",
                                                                            ".libs",
                                                                            NULL);
-  gchar *ld_library_path = g_getenv ("LD_LIBRARY_PATH");
+  const gchar *ld_library_path = g_getenv ("LD_LIBRARY_PATH");
```

Comment:
You've fixed this in https://github.com/endlessm/eos-updater/pull/242 too :)

Reply:
So I did, 9 months ago.